model_servers/object_detection_python/src/object_detection_server.py

```diff
@@ -9,7 +9,9 @@
 
 
 app = FastAPI()
-model = os.getenv("MODEL_PATH", default="facebook/detr-resnet-101")
+models_path = os.getenv("MODELS_PATH", default="/models")
+model_name = os.getenv("MODEL_NAME", default="facebook/detr-resnet-101")
+model = os.path.abspath(f"{models_path}/{model_name}")
```

Comment:
Please revert this file.  This `model` path will not  be compatible with changes being made in #256